x-pack/plugins/uptime/public/components/monitor/ml/manage_ml_job.tsx

```diff
@@ -87,7 +84,6 @@ export const ManageMLJobComponent = ({ hasMLJob, onEnableJob, onJobDelete }: Pro
           'data-test-subj': anomalyAlert
             ? 'uptimeDisableAnomalyAlertBtn'
             : 'uptimeEnableAnomalyAlertBtn',
-          icon: <EuiIcon type={anomalyAlert ? 'bellSlash' : 'bell'} size="m" />,
```

Comment:
no icons? :cry:

Reply:
Yes, that is my proposal. I added a few screenshots so the team can see what this looks like. cc @justinkambic